assets/locales/en/translation.json

```diff
@@ -215,9 +215,8 @@
   "Perfecto Token": "Perfecto Token",
   "Your Kobiton Username": "Your Kobiton Username",
   "Kobiton Access Key": "Kobiton Access Key",
-  "Headspin Host": "Headspin Host",
-  "Headspin Port": "Headspin Port",
-  "Headspin API Token": "Headspin API Token",
+  "serverTabHeasdpinWebDriverURL": "HeadSpin Web Driver URL",
+  "sessionHeadspinInvalidWebDriverURLError": "invalid HeadSpin Web Driver URL. Please make sure the Web Driver URL",
```

Comment:
The HeadSpin WebDriver URL is not valid. Please enter the correct URL it and try again.